lib/themes/dosomething/paraneue_dosomething/js/components/onboard.js

```diff
@@ -0,0 +1,21 @@
+const React = require('react');
+const ReactDOM = require('react-dom');
+
+import Carousel from './carousel.js';
+
+const Onboard = React.createClass({
+  render: function() {
+    return (
+      <div className="onboard">
+        <Carousel />
+      </div>
+    );
+  }
+});
+
+if (localStorage.getItem('dosomething_enable_onboarding') == "true") {
```

Comment:
Just a note... @katiecrane already started adding user information to `Drupal.settings` for the organ donation work we were doing...I think we should try to consolidate them as to not have two user objects...we really just need one so that anything can use it as needed. https://github.com/DoSomething/phoenix/blob/973e7cbeaf72abf05bd16c30cc7851d2e75b080e/lib/modules/dosomething/dosomething_campaign/dosomething_campaign.theme.inc#L222